apps/web/pages/api/v1/teams/[teamId]/add_demo_product/index.ts

```diff
@@ -0,0 +1,365 @@
+      include: {
+        environments: true,
+      },
+    });
+
+    type ProductWithEnvironment = Prisma.ProductGetPayload<typeof productWithEnvironment>;
+
+    const demoProduct: ProductWithEnvironment = await prisma.product.create({
+      data: {
+        name: "Demo Product",
+        team: {
+          connect: {
+            id: teamId,
+          },
+        },
+        environments: {
+          create: [
+            {
+              type: "production",
+              eventClasses: {
+                create: [
+                  {
+                    name: "New Session",
+                    description: "Gets fired when a new session is created",
+                    type: "automatic",
+                  },
+                  {
+                    name: "Exit Intent (Desktop)",
+                    description: "A user on Desktop leaves the website with the cursor.",
+                    type: "automatic",
+                  },
+                  {
+                    name: "50% Scroll",
+                    description: "A user scrolled 50% of the current page",
+                    type: "automatic",
+                  },
+                ],
+              },
+              attributeClasses: {
+                create: [
+                  {
+                    name: "userId",
+                    description: "The internal ID of the person",
+                    type: "automatic",
+                  },
+                  {
+                    name: "email",
+                    description: "The email of the person",
+                    type: "automatic",
+                  },
+                ],
+              },
+            },
+            {
+              type: "development",
+              eventClasses: {
+                create: [
+                  {
+                    name: "New Session",
+                    description: "Gets fired when a new session is created",
+                    type: "automatic",
+                  },
+                  {
+                    name: "Exit Intent (Desktop)",
+                    description: "A user on Desktop leaves the website with the cursor.",
+                    type: "automatic",
+                  },
+                  {
+                    name: "50% Scroll",
+                    description: "A user scrolled 50% of the current page",
+                    type: "automatic",
+                  },
+                ],
+              },
+              attributeClasses: {
+                create: [
+                  {
+                    name: "userId",
+                    description: "The internal ID of the person",
+                    type: "automatic",
+                  },
+                  {
+                    name: "email",
+                    description: "The email of the person",
+                    type: "automatic",
+                  },
+                ],
+              },
```

Comment:
and can we write this once and just call the variable in both the places instead of writing it twice?